src/newItinerary/HiddenStops.unit.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { fireEvent, render, screen } from '@testing-library/react'
+
+import { HiddenStop } from './HiddenStop'
+import { HiddenStops } from './HiddenStops'
+
+describe('HiddenStops', () => {
+  it('Should render the list item & allow stops toggle', () => {
+    render(
+      <HiddenStops label="2 stops">
+        <HiddenStop label="Paris" />
+        <HiddenStop label="Lyon" />
+      </HiddenStops>,
+    )
+    expect(screen.getByText('2 stops')).toBeInTheDocument()
+    expect(screen.getByRole('button')).toBeInTheDocument()
+
+    // Toggle stops.
+    // Note: aria-ignore makes .toBeInTheDocument() return false
```

Comment:
But you aren't using `aria-ignore` anymore, are you?

Reply:
`aria-hidden`*